### DISCUSSION TITLE
Android: Can't edit issue/pr labels, assignees, ...

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#3970] [Mobile]
I can't edit labels, assignees, projekts, reviewers on PRs/issues in the android app for github.

The labels, ... are shown for a short time in discussion but won't really set. After relaod the view they are lost.

I've heard this is on the roadmap. Is there any time when this will be fixed? It's really annoying!!!!

**Comment by elisealix22** [comment]
@htcfreek do you happen to have any video or screenshot available to show us the bug you're running into?

If you open an issue or pull request and swipe up on the bottom sheet to edit labels for example, you should go to a new screen where you can add or remove any label. After making your changes, make sure to tap the `Save` button in the upper right hand of the screen. Then if you return back to the main issue or pull request, you should be able to refresh and notice your change was saved. Let me know if this is still an issue for you and I'll dig in further!

**Reply by htcfreek** [reply]
Sory. No.

1. Click on edit link near labels.
2. Chose changes and click save.
3. Changes shown on discussion. But not in details near edit link.
4. Relaod issue/PR: Changes not shown in discussion anymore.


**NOTE:**
But I don't have full rights on the repo. So maybe this is the cause. I additional can only edit them on my own issues/PRs. (One the webpage I can edit them on all issues/PRs in the repo.)

**Reply by elisealix22** [reply]
Ahhhh interesting! Thank you for the additional info. I filed an issue for our team and we will look into it. If you happen to know of a public repo where it is happening that you can share, please send it our way.

**Reply by htcfreek** [reply]
> Ahhhh interesting! Thank you for the additional info. I filed an issue for our team and we will look into it. If you happen to know of a public repo where it is happening that you can share, please send it our way.

Microsoft/PowerToys

Have you wrote in the issues that this happened on reviews, labels, assignees, and others?

**Reply by Jay-o-Way** [reply]
I wanna ask for the same abilities. Just like @htcfreek, I have triage permits for the PowerToys repo. Been using the browser for a while on my Android phone, but today I have the app. (Quite difficult to find via the website, btw)
Me too can't set labels or close an issue (etc.) something that obviously works fine in the browser. My special powers are useless here 😟
(note: I **can** set labels for my **own** issues, but not for others)

**Comment by fflaten** [comment]
Same on iOS (for a long time). Add label or close issue... feel the haptic response and it's reverted. Guessing it's related to triage-role not working in the mobile app. My triage-role works fine on the website.

**Reply by htcfreek** [reply]
Yes. Seems to be related to your assigned role.

**Reply by elisealix22** [reply]
Thank you so much for the additional information. This really helps! We're looking into it.

**Reply by htcfreek** [reply]
@elisealix22 
Great. Can you please inform us on status updates. Thank you in advice.

**Reply by htcfreek** [reply]
@elisealix22 
Any new state updates?

**Reply by alcere** [reply]
Hi @htcfreek @fflaten, thanks for following up on this.
I've verified this bug on iOS & Android, and I've created issues in both platforms to track this work and expect this behavior to be fixed in a future release. 
We are sorry for the trouble.

**Reply by Jay-o-Way** [reply]
@alcere any news please?

**Reply by alcere** [reply]
Hi @Jay-o-Way, sadly, this issue is still under investigation :( It might take some time as fixing the root cause implies a few changes from different teams. We will keep you updated. Sorry again for the inconvenience!

**Reply by huuduyentinh14020201** [reply]
> Hi @Jay-o-Way, sadly, this issue is still under investigation :( It might take some time as fixing the root cause implies a few changes from different teams. We will keep you updated. Sorry again for the inconvenience!

**Comment by Abrahimoopp** [comment]
109.161.212.133

**Reply by mmcelwain01** [reply]
I appreciate all of your hard work. I am hoping this all gets worked out quickly as well. GitHub is the absolute best!!

**Comment by htcfreek** [comment]
For me it's fixed for issue/pr labels. Can someone confirm that it's fixed for assignments too?

**Reply by Jay-o-Way** [reply]
Not yet
![Screenshot_20221121-004640.png](https://user-images.githubusercontent.com/65828559/202933150-e0ed96fe-f514-400f-9ae9-f2ecc5f65a4a.png)

**Reply by larseggert** [reply]
For me, assignments are still broken on the iOS app. Annoying.

**Reply by Jay-o-Way** [reply]
Can't close an issue either.  😖

**Reply by alcere** [reply]
Hi! we are still (slowly) working on this issue, sorry for the troubles!
@Jay-o-Way could you please share more information about the close issue problem on Android? Thanks!

**Reply by Jay-o-Way** [reply]
I have triage permissions for one repo. Message in the app says I don't. @alcere

**Reply by alcere** [reply]
Thanks @Jay-o-Way! I can reproduce the issue. We will work on a fix asap.

**Reply by alcere** [reply]
Hi! We are sorry for the long wait :(. Android Beta 1.104.0 should allow triage role users to edit assignees. The iOS fix will eventually arrive too.

**Comment by Jay-o-Way** [comment]
@alcere Does **closing** an issue fall within the scope of this discussion too? Have triage rights to a repo that isn't my own.
![Screenshot_20230311-122807~2.png](https://user-images.githubusercontent.com/65828559/224482699-a294dcac-2199-4d57-bf4d-827e72e645d3.png)

**Reply by htcfreek** [reply]
And I have the same on the same repository with reopening issues.

**Reply by alcere** [reply]
@Jay-o-Way @htcfreek Thanks for asking! That issue is on our radar, but unfortunately, it might take some time for a fix to arrive :(.

**Comment by ghost** [comment]
@aconsuegra

**Comment by willcl-ark** [comment]
Any updates on this? Still issues with iOS app for a repo I have triage permissions for...

**Reply by aconsuegra** [reply]
Hey @willcl-ark, this was supposed to be fixed some time ago, and we haven't been able to reproduce it anymore on our end. What version of the app are you using? Could you elaborate on the repro steps so we are exactly on the same page?

**Reply by willcl-ark** [reply]
I am using current version of the app on iOS 1.151.1, many updates later, but this still doesn't work

![image](https://github.com/community/community/assets/6606587/427afecf-136d-4272-9bc0-6a0c8130c567)

Will it ever be fixed?

**Reply by willcl-ark** [reply]
To be clear repo steps are:

Log in, try to close pull in repo I have triage perms on, and can close just fine on desktop/browser. See error

**Reply by aconsuegra** [reply]
Thanks for following up! Do other things like adding labels or assignees work? Or you are experiencing this issue only while trying to close a PR?

**Reply by willcl-ark** [reply]
Just tested adding labels to an issue and they work ok. So perhaps just closing pull requests?

**Reply by aconsuegra** [reply]
Super useful info, thanks for testing!

**Reply by htcfreek** [reply]
Can confirm this behavior for Android. Only closing fails.

**Reply by aconsuegra** [reply]
@willcl-ark Just wanted to let you know that this issue should be fixed now on iOS (no need to update the app). @htcfreek This is also fixed in the last Android beta and the fix will be in our production build next week

**Reply by willcl-ark** [reply]
Thanks, I'll let you know how it goes when I next try to close.

**Reply by willcl-ark** [reply]
It's working now, thanks!

**Comment by ghost** [comment]
@aconsuegra I am also experiencing this on iOS.

I cannot edit any issue I have created within the app.

I have to open the issue in Safari to edit the issue.

**Reply by aconsuegra** [reply]
Thanks for reaching out @jslawler. A few questions to help us debug the issue:

- Does it matter in which repository you create the issue? Or do you experience this in all the repositories?
- When you say "edit any issue", do you mean editing properties or an issue like labels or assignees? Or editing the title/body? Both?

Thanks in advance.

**Reply by ghost** [reply]
> Thanks for reaching out @jslawler. A few questions to help us debug the issue:
> 
> 
> 
> - Does it matter in which repository you create the issue? Or do you experience this in all the repositories?
> 
> - When you say "edit any issue", do you mean editing properties or an issue like labels or assignees? Or editing the title/body? Both?
> 
> 
> 
> Thanks in advance.

Appreciate the swift reply.

This appears to occur on all title and body edits, tested across 2 seperate repo's for 2 different companies, Transmission and AdGuard.

It doesn't seem to affect repo's that I own though.

**Comment by giosueDelgado** [comment]
We have the same issue, if I assign the Triage role to a user he cannot manage the description or flag the to-do list
